packages/skeleton/src/base/theme.scss

```diff
@@ -134,3 +134,65 @@
 	--text-9xl: calc(8rem * var(--text-scaling));
 	--text-9xl--line-height: calc(1 * var(--text-scaling));
 }
+
+/* Add missing theme utilities - solution until variables can be renamed to follow Tailwind theme namespaces in v4 */
```

Comment:
If these are `@utility` classes, they should live in the `/utilities` section of the package. Not in `/base/theme.css`